lib/Runtime/Library/TypedArray.cpp

```diff
@@ -1730,6 +1730,24 @@ namespace Js
         return JavascriptArray::CopyWithinHelper(nullptr, typedArrayBase, typedArrayBase, length, args, scriptContext);
     }
 
+    Var TypedArrayBase::GetKeysEntriesValuesHelper(Arguments& args, ScriptContext *scriptContext, LPCWSTR apiName, JavascriptArrayIteratorKind kind)
```

Comment:
Sure.

---
In reply to: [79928194](https://github.com/Microsoft/ChakraCore/pull/1618#discussion_r79928194) [](ancestors = 79928194,79927525)